views/lib/components/table.hbs

```diff
@@ -32,21 +33,23 @@
                                     {{#each this}}
                                         {{#if this.method}}
                                             <form action="{{this.link}}" method="post" class="inline">
-                                                <input type="hidden" name="_method" value="{{this.method}}"/>
+                                                <label for="method" hidden>{{this.label}}</label>
+                                                <input id="method" type="hidden" name="_method" value="{{this.method}}" hidden />
                                                 {{> "lib/components/csrfInput"}}
-                                                <button type="submit" class="btn btn-secondary btn-sm no-inline-styling {{this.class}}" title="{{this.title}}">
+                                                <button type="submit" class="btn btn-secondary btn-sm no-inline-styling {{this.class}}" title="{{this.title}}" aria-label="{{this.title}}" tabindex="0">
```

Comment:
do we need the title when we have the aria? :)

Reply:
It depends on whether we need a tooltip. If not, then aria-label should be sufficient.